decidim-core/app/controllers/decidim/profiles_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Decidim
+  # The controller to handle the user's public profile page.
+  class ProfilesController < Decidim::ApplicationController
+    skip_authorization_check
+
+    helper Decidim::Messaging::ConversationHelper
+
+    helper_method :user
+
+    def show; end
```

Comment:
Can we skip action declaration or do you prefer being explicit?

Reply:
I prefer being explicit but would that actually work anyways? :thinking:

Reply:
Yup, if Rails fins a `show`file it would render it

Reply:
Ha, interesting! So as long as I define the route and the template, I don't need the controller action, right? Wow :)

Reply:
Yup!